src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.cs

```diff
@@ -193,7 +193,7 @@ public int IndexOf(T item, int startIndex, int count, IEqualityComparer<T>? equa
         public int LastIndexOf(T item)
         {
             var self = this;
-            if (self.Length == 0)
+            if (self.IsEmpty)
```

Comment:
No strong opinions either way, but what motivated you to use `IsEmpty` instead of `self.Length == 0`?

Reply:
Because I unintentionally found there was place to use `IsEmpty` and checked implementation of `IsEmpty `and `Length == 0`, they are same at least from c# code path (including call stack depth). So I just replaced to use it as its readability.